lib/active_model_serializers/adapter/json_api/relationship.rb

```diff
@@ -25,7 +25,7 @@ def as_json
           meta = meta_for(association)
           hash[:meta] = meta if meta
```

Comment:
My thought was this is a placeholder for other things we might want to do, but sure no worries.